payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/ExternalPaymentMethodsSerializer.kt

```diff
@@ -0,0 +1,25 @@
+package com.stripe.android.ui.core.elements
+
+import androidx.annotation.RestrictTo
+import androidx.annotation.WorkerThread
+import kotlinx.serialization.json.Json
+import kotlinx.serialization.serializer
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+internal object ExternalPaymentMethodsSerializer {
+
+    private val format = Json {
+        ignoreUnknownKeys = true
+        classDiscriminator = "#class"
+
+        // needed so that null values in the spec are parsed correctly
```

Comment:
Is there a test for this? I wouldn't expect us to need this.

Reply:
yeah, we also don't need this. Removed